src/adapter/src/explain_new/mod.rs

```diff
@@ -287,20 +284,18 @@ where
     }
 }
 
-impl<'a> DisplayText<RenderingContext<'a>> for Displayable<'a, LinearOperator> {
+impl<'a> DisplayText<RenderingContext<'a>> for Displayable<'a, MapFilterProject> {
```

Comment:
can you just copy-paste the code from the 
```rust
impl<'a> DisplayText<PlanRenderingContext<'_, Plan>> for Displayable<'a, MapFilterProject> {

}
```
definition here for now? It won't require any modifications, and I'll open a follow up PR to make the definition generic over the rendering context parameter next week.

Reply:
Yes, but I'll roll it up in some other output changes. Is there a better way to grind through all of the explain CI rewrites other than `sqllogictest --rewrite-results` and by hand for `testdrive`?